packages/firebase_vertexai/firebase_vertexai/example/pubspec.yaml

```diff
@@ -0,0 +1,40 @@
+name: Vertex AI for Firebase Example
```

Comment:
```suggestion
name: vertex_ai_example
```

Reply:
One more thing actually - I think @Lyokone wanted changes to be reverted to the app check example. Essentially, all the files in firebase_app_check/**/* as they aren't part of the PR. Thanks 🙏